server/src/app.service.ts

```diff
@@ -49,10 +49,45 @@ export class GameService {
   }
 
   getGame(id: string): Game {
-      if(this.games[id]) {
-        return this.games[id];
-      } else {
+    if(this.games[id]) {
+      return this.games[id];
+    } else {
       throw new NotFoundException('Not Found', 'There is no such game')
     }
   }
+
+  enterTheGame(id: string, side: string): NewGameResonse {
+    let game: Game;
+
+    if(this.games[id]) {
```

Comment:
let game = this.games[id];

    if(!game) {
      throw new NotFoundException('Not Found', 'There is no such game')
    }